source/modules/gestion/certificates.rst

```diff
@@ -0,0 +1,71 @@
+Gérer les certificats
+=====================
+
+Au sein de GLPI, il est possible de matérialiser les certificats.
+
+Cette gestion a pour objectif de fournir:
+
+- un inventaire de tous les certificats de l'organisation;
```

Comment:
pour faire une liste (`<ul><li>`), la syntaxe rst est ` * un point` (http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks)